audio_play/src/audio_play.cpp

```diff
@@ -129,7 +129,7 @@ namespace audio_transport
 
 int main (int argc, char **argv)
 {
-  ros::init(argc, argv, "audio_play");
+  ros::init(argc, argv, "audio_play", ros::init_options::AnonymousName);
```

Comment:
Do you mind, giving me the rationale behind this modification ?

Reply:
http://wiki.ros.org/roscpp/Overview/Initialization%20and%20Shutdown#Initializing_the_roscpp_Node
> node_name 
> This is the name that will be assigned to your node unless it's overridden by one of the remapping arguments. Node names must be unique across the ROS system. If a second node is started with the same name as the first, the first will be shutdown automatically. **In cases where you want multiple of the same node running without worrying about naming them uniquely, you may use the init_options::AnonymousName option described below.**

http://wiki.ros.org/roscpp/Overview/Initialization%20and%20Shutdown#Initialization_Options
> ros::init_options::AnonymousName
> Anonymize the node name. Adds a random number to the end of your node's name, to make it unique.

Reply:
In case someone else forgets to rename the node

Reply:
But does that impact the calling of the node when trying to communicate with it ? If it adds a random number, would it be impossible to know what will be the next number when you'll try to communicate with it ?

Reply:
We communicate with topics, it doesn't matter what the node name is. At least not here.

Reply:
ALright thank you